.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb-base"
```

Comment:
Great choice! I'm trying use to Airbnb rules as well with ESLinter. But I've read a tutorial about using ESLinter and Prettier together. ESLinter is for linting and Prettier is for code formatting. I'll share the link here :slightly_smiling_face: 
[Configure Prettier and ESLint in VSCode](39digits.com/configure-prettier-and-eslint-in-visual-studio-code/)